src/pages/bioclimatic/component.jsx

```diff
@@ -1,37 +1,80 @@
-import React, { useState } from 'react';
+import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
-import { LayerManager, Layer } from 'layer-manager/dist/components';
-import { PluginMapboxGl } from 'layer-manager';
+// import { LayerManager, Layer } from 'layer-manager/dist/components';
```

Comment:
are these comments for later use?

Reply:
No, we don't need them anymore, removed. Thanks :+1: